techniques/html/H88.html

```diff
@@ -96,50 +92,50 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
+					<strong>仕様で定められている機能だけを用いる:</strong> HTML では、ウェブページで用いることのできる要素、属性及び属性値の一式を定義している。それらには固有のセマンティックな意味を持ち、ユーザエージェントが特定の方法で処理するよう意図されている。しかし、時には、追加の機能が一般的な制作方法になることがある。これらは通常、一つのユーザエージェントだけによってサポートされる。仕様にはない方法を用いた場合、多くのユーザエージェントはそれを当分の間サポートしなかったり、いつまでもサポートしなかったりする。さらに、これらの機能の使用に関する標準仕様がないため、ユーザエージェントによってそのサポート方法が異なることもある。そして、主流のユーザエージェントと比べ少ないリソースで開発されている支援技術は、有用なサポートを追加する場合でも時間を要するため、アクセシビリティに影響を及ぼすことになる。よって、予期しないアクセシビリティの問題を回避するためにも、コンテンツ制作者は HTML で定義されていない機能を避けるべきである。</li>
          <li>
-					<strong>Using features in the manner prescribed by the specification</strong> The HTML specification provides specific guidance about how particular elements, attributes, and attribute values are to be processed and understood semantically. Sometimes, however, authors use features in a manner that is not supported by the specification, for example, using semantic elements to achieve visual effects without intending the underlying semantic message to be conveyed. This leads to confusion for user agents and assistive technologies that rely on correct semantic information to present a coherent representation of the page. It is important to use HTML features only as prescribed by the HTML specification.</li>
+					<strong>仕様によって規定されている方法で機能を用いる:</strong> HTML の仕様では、どのように特定の要素、属性及び属性値をセマンティックに処理し解釈すべきかについて、具体的なガイダンスを提供している。しかし、時には、コンテンツ制作者が仕様ではサポートされていない方法でそれらを用いることがある。例えば、本来のセマンティックなメッセージが伝わることを意図せずに、セマンティックな要素を用いて視覚的な効果を得ることが挙げられる。これは、正確なセマンティック情報に基づいて首尾一貫したページを提示しているユーザエージェント及び支援技術に対して混乱を招くことにつながる。あくまで HTML の仕様によって規定されている機能として、HTML の機能を用いることが重要である。</li>
          <li>
-					<strong>Making sure the content can be parsed</strong> HTML defines how content should be encoded in order to be correctly processed by user agents. Rules about the structure of start and end tags, attributes and values, nesting of elements, etc. ensure that user agents will parse the content in a way to achieve the intended document representation. Following the structural rules in these specifications is an important part of using these technologies according to specification. </li>
+					<strong>コンテンツが解析可能であることを確認する:</strong> HTML では、ユーザエージェントによって正しく処理されるためには、コンテンツをどのようにコーディングすべきかも定めている。開始タグ及び終了タグ、属性及び値、要素の入れ子などの構造に関するルールは、意図された文書の表現となるようにユーザエージェントがコンテンツを解析できるようにするためにある。HTML の仕様における構造のルールに従うことが、仕様に準じてそれらのウェブコンテンツ技術を用いる上で重要なことの一つである。</li>
       </ul>
 		</section>
 		
-	 <section id="resources"><h2>Related Resources</h2>
-<p style="margin-bottom: 1.5em;"><em><small>No endorsement implied.</small></em></p>
+	 <section id="resources"><h2>関連リソース</h2>
+<p style="margin-bottom: 1.5em;"><em><small>推奨を意味するものではない。</small></em></p>
 
 
       
-         <p>Refer to the resources section of <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G134">Validating Web
-                        pages</a>. </p>
+         <p><a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G134">Validating Web
```

Comment:
ここのリンク URL は、日本語訳されたテクニック集の G134 にするのがよいかと思います。
```suggestion
         <p><a href="../general/G134">Validating Web
```